scripts/java-api-check-all.sh

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env bash
 
-set -Eeuxo pipefail
+set -Eeuo pipefail
 # Usage:
 #   ./java-api-check-all.sh <current release tag (empty for branches)>
```

Comment:
Can we add a note to build the project before running `java-api-check-all.sh` while running locally?